### ISSUE TITLE
add option to redirect output to stdout

### ISSUE THREAD

**Issue by xoviat** [arun11299/cpp-subprocess#67]
In python, it can be directed by passing stdout.

**Comment by zzm3145** [comment]
Agreed. Redirecting to stdout & stderr is the default behavior in Python subprocess.

**Comment by arun11299** [comment]
@xoviat Would it be possible for you to create a PR for this ?
